Plutarch/Api/V1/Value.hs

```diff
@@ -123,6 +124,11 @@ newtype PValue (keys :: KeyGuarantees) (amounts :: AmountGuarantees) (s :: S)
   deriving anyclass (PlutusType, PIsData, PShow)
 instance DerivePlutusType (PValue keys amounts) where type DPTStrat _ = PlutusTypeNewtype
 
+instance PShow (PValue keys amounts) where
```

Comment:
Why not just derive this?

Reply:
Maybe you're unaware of #537 ?

Reply:
> Maybe you're unaware of #537 ?

I was unaware indeed. I removed the related commits from this PR.